Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -246,6 +246,7 @@ extension _RSA.Signing {
         internal enum Backing {
             case pkcs1v1_5
             case pss
+            case pssZero
```

Comment:
Can we ensure that other callers would not use this by adding a comment on this enum case or having a more deterring name?

Reply:
This is an internal backing enum and not exposed in the API. Are you suggesting we go further and annotate it in some fashion to deter accidental _internal_ usage?

Reply:
Better safe than sorry, because deterministic signatures are not at all desirable in many other settings.